server_installation/topics/operator/keycloak-cr.adoc

```diff
@@ -28,7 +28,10 @@ The Keycloak custom resource is a YAML file that defines the parameters for inst
 ifeval::[{project_community}==true]
  or an Ingress for Kubernetes
 endif::[]
- for the {project_name} cluster.
+ for the {project_name} cluster. You can set `host` to override automatically chosen host name for Route
```

Comment:
Yes, thank you.  I had meant to say add "the" BEFORE "automatically."  thanks for checking.